icons/starship.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><title>Starship icon</title><path d="M11.01 23.9776c0-.022.0481-.0848.107-.1396.168-.1565.5759-.6751.8407-1.069.3082-.4584.6037-1.0117.8347-1.563.3641-.869.7505-2.2112.661-2.2957-.0392-.037-1.4586-.5084-2.5407-.844-.4884-.1514-.6231-.1795-.6708-.1399-.0329.0273-.1442.1804-.2475.3404-.428.6627-1.3466 1.6071-1.9613 2.0161-.9241.615-1.953 1.0208-3.0207 1.1916l-.385.0616-.3757-.3162C2.297 19.574.9178 17.3459.3007 14.8365c-.2769-1.1263-.2878-1.2341-.2884-2.837C.012 10.5652.014 10.529.1291 9.923c.4673-2.459 1.5077-4.4976 3.1828-6.2367C5.2432 1.6813 7.7243.4305 10.4754.075c.9795-.1266 2.5032-.0894 3.536.0865 3.4338.5846 6.444 2.6418 8.2487 5.6371 1.9505 3.2373 2.2676 7.1926.8583 10.7066-.4042 1.0078-.9262 1.922-1.618 2.8337-.3928.5177-1.5878 1.7213-2.1026 2.1177-1.7355 1.3366-3.743 2.1817-5.8592 2.4665-.454.0611-2.5287.106-2.5287.0546zm3.6467-8.0926l.2136-.5345.1565.0068c.086.0037.7075.1294 1.3808.2793.6734.1498 1.2424.2613 1.2644.2477.0984-.0608.0039-1.1463-.1598-1.8346-.1818-.7649-.5334-1.549-.895-1.9955-.0763-.0943-.1387-.2064-.1387-.2493 0-.0428.0564-.2432.1254-.4453.2257-.6618.289-.9892.3134-1.6216.0345-.8923-.0764-1.4378-.4564-2.244-.323-.685-1.1565-1.9088-1.3002-1.9088-.16 0-1.3076.937-2.0371 1.6635-1.052 1.0474-1.5241 1.9453-1.611 3.0634l-.0298.3836-.5955.3266c-1.0605.5816-1.6478 1.0097-2.1986 1.6026-.1706.1836-.3102.3465-.3102.3619 0 .0234 2.246.9546 2.6004 1.0782.0624.0217.113.07.1128.1074-.0002.0373-.065.298-.1439.5792-.0789.2812-.1434.542-.1434.5796 0 .0774-.0422.0634 1.9119.6354 1.6914.495 1.5936.4681 1.6652.46.0384-.0045.1425-.2092.2752-.5416zM6.76 7.718c.2105-.469.701-.8226 1.4156-1.0207.4156-.1151.4155-.1145.0178-.2069-.4944-.115-.9934-.4566-1.2466-.8537-.1381-.2167-.2169-.4061-.3176-.7638-.0371-.1318-.0763-.2307-.0871-.2199-.0108.0108-.0476.1278-.0819.26-.1655.6394-.5555 1.1511-1.0578 1.3879-.161.0759-.4205.1714-.5766.2123l-.2838.0744.272.0918c.8514.2872 1.3598.7808 1.6163 1.5693l.1142.3509.0543-.3217c.0299-.177.1024-.429.1612-.56zm7.228 2.8267c-.6619-.3082-.7901-1.226-.2389-1.71.4425-.3885 1.0264-.3667 1.435.0537.227.2334.3078.4311.3072.7516-.0008.3781-.2851.7966-.6429.946-.2171.0908-.618.0715-.8604-.0413z"/></svg>
```

Comment:
This version is still showing some differences... :thinking:

I'll take a look and see if I'm able to get a path made up for this...